packages/flutter/lib/src/widgets/view.dart

```diff
@@ -106,13 +146,593 @@ class View extends StatelessWidget {
+  /// responsible for managing the render tree of that view. If there is no
+  /// [View] ancestor, [RendererBinding.rootPipelineOwner] is returned instead.
+  static PipelineOwner pipelineOwnerOf(BuildContext context) {
+    return context.dependOnInheritedWidgetOfExactType<_PipelineOwnerScope>()?.pipelineOwner
+        ?? RendererBinding.instance.rootPipelineOwner;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return RawView._deprecated(
+      view: view,
+      deprecatedPipelineOwner: _deprecatedPipelineOwner,
+      deprecatedRenderView: _deprecatedRenderView,
+      builder: (BuildContext context, PipelineOwner owner) {
+        return _ViewScope(
+          view: view,
+          child: _PipelineOwnerScope(
+            pipelineOwner: owner,
+            child: MediaQuery.fromView(
+              view: view,
+              child: child,
+            ),
+          ),
+        );
+      }
+    );
+  }
+}
+
+/// A builder for the content [Widget] of a [RawView].
+///
+/// The widget returned by the builder defines the content that is drawn into
+/// the [FlutterView] configured on the [RawView].
+///
+/// The builder is given the [PipelineOwner] that the [RawView] uses to manage
+/// its render tree. Typical builder implementations make that pipeline owner
+/// available as an attachment point for potential child views.
+///
+/// Used by [RawView.builder].
+typedef RawViewContentBuilder = Widget Function(BuildContext context, PipelineOwner owner);
+
+/// The workhorse behind the [View] widget that actually bootstraps a render
+/// tree.
+///
+/// It instantiates the [RenderView] as the root of that render tree and adds it
+/// to the [RendererBinding] via [RendererBinding.addRenderView]. It also owns
+/// the [PipelineOwner] that manages this render tree and adds it as a child to
+/// the surrounding parent [PipelineOwner] obtained with [View.pipelineOwnerOf].
+/// This ensures that the render tree bootstrapped by this widget participates
+/// properly in frame production and hit testing.
+///
+/// The [RawView] widget faces the same limitations in terms of where it can
+/// appear in the widget tree as the [View] widget. See the [View] widget for
+/// details.
+///
+/// The [RawView] widget is rarely used directly. Instead, consider using the
+/// [View] widget, which also inserts a proper [MediaQuery] for the [view] into
+/// the tree and injects the [PipelineOwner] of this view into the tree for
+/// potential child views.
+class RawView extends RenderObjectWidget {
+  /// Create a [RawView] widget to bootstrap a render tree that is rendered into
+  /// the provided [FlutterView].
+  ///
+  /// The content rendered into that [view] is determined by the [Widget]
+  /// returned by [builder].
+  RawView({
+    required this.view,
+    required this.builder,
+  }) : _deprecatedPipelineOwner = null,
+       _deprecatedRenderView = null,
+       super(key: GlobalObjectKey(view));
+
+  RawView._deprecated({
```

Comment:
Why is `RawView._deprecated` used exclusively in production code, while `RawView()` is used throughout the unit tests?

Reply:
This is obsolete since with RenderView becoming private I merged the two constructors. However for completeness: the reason for this is: The only usage of rawview within the framework so far is to make the implicit view work with the renderView and pipelineOwner from the binding and that needs the deprecated constructor. There's currently no other usage of views in the framework.

There were also tests that used the deprecated constructor implicitly: Any test that didn't use an explicit RawView was falling back to the deprecated constructor here used by the bindings wrapInDefaultView method...

Reply:
Anyway now that `_RawView` is private this is outdate :)